ReactSkia/sdk/NopollWebsocket.cpp

```diff
@@ -0,0 +1,298 @@
+/*
+* Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+#include <nopoll.h>
+
+#include "ReactSkia/utils/RnsLog.h"
+#include "NopollWebsocket.h"
+
+using namespace std;
+namespace facebook {
+namespace react {
+
+std::queue<NopollRequest*> RequestQueue;
+int currentWorkerThreadCount = 0;
```

Comment:
Can we make this as part of singleton class object (static member may be) ? Same for other global variables.

Reply:
moved all global variables to private